frontend/src/components/bs/components/spaces-table/components/action-cell/action-cell.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconButton } from 'components/common/common';
+import { IconName } from 'common/enums/enums';
+
+const ActionCell = (
+  id: string,
+  onSpaceDelete: (id: string) => void,
+): JSX.Element => {
+  return (
+    <IconButton icon={IconName.TRASH} onClick={(): void => onSpaceDelete(id)} />
```

Comment:
avoid anonymous functions

Reply:
According to previous suggestions, IconButton needs just a callback in props, how then I pass it?
![image](https://user-images.githubusercontent.com/86227025/157106863-9bab9cba-1efa-40f5-ae50-2d7e0ad55c21.png)

Reply:
?
```
  onSpaceDelete: (id: string) => void,
): JSX.Element => {
  const handleDelete = () => onSpaceDelete(id);

  return (
    <IconButton icon={IconName.TRASH} onClick={handleDelete} />
```